wgsl/index.bs

```diff
@@ -3843,42 +3846,23 @@ Each pipeline input or output is one of:
 A <dfn noexport>built-in input variable</dfn> provides access to system-generated control information.
 The set of built-in inputs are listed in [[#builtin-variables]].
 
-To declare a variable for accessing a particular input built-in *X*:
+To declare a variable for accessing a particular input built-in *X* from an entry point:
 
-* Declare a module-scope variable in the [=storage classes/in=] storage class,
+* Declare a parameter of the entry point function,
     where the [=store type=] is the listed store type for *X*.
-* Apply a `builtin(`*X*`)` attribute to the variable.
-* The variable must not have an initializer. The system provides the value.
+* Apply a `builtin(`*X*`)` attribute to the parameter.
 
 A <dfn noexport>built-in output variable</dfn> is used by the shader to convey
 control information to later processing steps in the pipeline.
 The set of built-in outputs are listed in [[#builtin-variables]].
 
-To declare a variable for accessing a particular output built-in *Y*:
-
-* Declare a module-scope variable in the [=storage classes/out=] storage class,
-    where the [=store type=] is the listed store type for *Y*.
-* Apply a `builtin(`*Y*`)` attribute to the variable.
-* The variable may have an initializer, or not, as described in [[#variables]].
-
-<div class='example wgsl global-scope' heading="Declaring built-in variables">
-  <xmp>
-    // vertex shader output builtin
-    [[builtin(position)]] var<out> my_position : vec4<f32>;
-
-    // fragment shader input builtin
-    [[builtin(frag_coord)]] var<in> coord : vec4<f32>;
-
-    // compute shader builtin
-    [[builtin(global_invocation_id)]] var<in> global_id : vec3<u32>;
-  </xmp>
-</div>
-
-The `builtin` attribute must not be applied to a variable in a storage class other than [=storage classes/in=] or [=storage classes/out=].
+To declare a variable for accessing a particular output built-in *Y* from an entry point:
 
-An input built-in must only be applied to a variable in the [=storage classes/in=] storage class.
+* Add a variable to the result of the entry point, where [=store type=] is the listed store type for *Y*.
```

Comment:
This reads as though I can have multiple return values. This should be clarified that you need to create a struct if there is more then one return value.